migrations/migrations/20210621204446-update-users-add-admin.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     queryInterface.addColumn('users', 'role', {
       allowNull: false,
       type: Sequelize.INTEGER,
-      defaultValue: 2,
+      defaultValue: 0,
```

Comment:
Instead of leaving magic numbers, you could use the role variables that you have in the constant files to set the default value